persistence-cassandra/javadsl/src/test/scala/com/lightbend/lagom/javadsl/persistence/cassandra/CassandraReadSideSpec.scala

```diff
@@ -44,3 +44,17 @@ class CassandraReadSideSpec extends CassandraPersistenceSpec(CassandraReadSideSp
   }
 
 }
+
+class CassandraReadSideAutoCreateSpec extends CassandraPersistenceSpec(CassandraReadSideSpec.noAutoCreateConfig) {
+  import system.dispatcher
+
+  private lazy val testSession: CassandraSession = new CassandraSession(system)
+  private lazy val testCasConfigProvider: CassandraProvider = new CassandraProvider(system)
+  private lazy val offsetStore = new JavadslCassandraOffsetStore(system, testSession, testCasConfigProvider, ReadSideConfig())
+
+  "ReadSide" must {
```

Comment:
I think the tests could be a bit more specific that it's `"A Cassandra Read-Side"`.